src/allotropy/allotrope/schemas/plate-reader/BENCHLING/2023/09/plate-reader.json

```diff
@@ -474,6 +480,53 @@
                     "$asm.pattern": "value datum",
                     "$asm.type": "http://www.w3.org/2001/XMLSchema#string",
                     "$ref": "#/$defs/tStringValue"
+                  },
+                  "image feature aggregate document": {
+                    "type": "object",
+                    "$asm.property-class": "TODO",
+                    "$asm.pattern": "aggregate datum",
+                    "required": [
+                      "image feature document"
+                    ],
+                    "properties": {
+                      "image feature document": {
+                        "type": "array",
+                        "$asm.array-ordered": true,
+                        "$asm.array-mixed": false,
+                        "$asm.pattern": "indexed datum",
+                        "items": {
+                          "type": "object",
+                          "$asm.property-class": "http://purl.allotrope.org/ontologies/result#AFR_0002659",
```

Comment:
@alejandro-salgado, image feature document will be a new term to AFO so property class should be "TODO".